source/documentation/administration_guide/topics/Replacing_the_Manager_CA_Certificate.adoc

```diff
@@ -6,7 +6,7 @@
 
 You can configure your organization's third-party CA certificate to authenticate users connecting to the {virt-product-fullname} {engine-name} over HTTPS.
 
-Using a third-party CA certificate for HTTPS connections does not affect the certificate used for authentication between the {engine-name} and hosts. They will continue to use the self-signed certificate generated by the {engine-name}.
+Using a third-party CA certificate for HTTPS connections does not affect the certificate used for authentication between the {engine-name} and hosts, or disk transfer URLs(link:{URL_rest_api_doc}index#types/image_transfer[transfer_url]). They will continue to use the self-signed certificate generated by the {engine-name}.
```

Comment:
Suggested rewrite:

Third-party CA certificates are not used for authentication between the {engine-name} and hosts or for link:{URL_rest_api_doc}index#types/image_transfer[disk transfer URLs]. These HTTPS connections use the self-signed certificate generated by the {engine-name}.